index.html

```diff
@@ -885,27 +1005,49 @@ <h4>Text</h4>
                   <code>tts:direction</code> attribute on <code>&lt;span&gt;</code> elements.
                   Similarly metadata can be added using attributes or <code>&lt;metadata&gt;</code> elements.</aside>
               </li>
-              <li>The <code>&lt;p&gt;</code> element MUST have one <a>Text Language Source</a> property.</li>
-              <li>The <code>&lt;p&gt;</code> element SHOULD have an <code>xml:lang</code> attribute corresponding to the language of the <a>Text</a> object.
-                <p class="note">If a <code>&lt;p&gt;</code> element omits the <code>xml:lang</code> attribute then its computed language
-                is derived by inheritance from its parent element, and so forth up to the root <code>&lt;tt&gt;</code> element,
-                which is required to set the <a>Primary Language</a> via its <code>xml:lang</code> attribute.
-                Care should be taken if changing the <a>Primary Language</a> of a <a>DAPT Script</a> in case
-                doing so affects descendant elements unexpectedly.
-                Authors can mitigate this risk by explicitly setting <code>xml:lang</code> on all <code>&lt;p&gt;</code> elements.</p>
+              <li>The <code>&lt;p&gt;</code> element SHOULD have a <a><code>daptm:langSrc</code></a> attribute
+                indicating whether the <a>Text</a> is <a>Original</a> or a <a>Translation</a> and if its
+                source had an inherent language.
+                <aside class="note">
+                  <p>If a <code>&lt;p&gt;</code> element omits the <a><code>daptm:langSrc</code></a> attribute
+                    then its computed value is derived by inheritance from its parent element,
+                    and so forth up to the root <code>&lt;tt&gt;</code> element.</p>
+                  <p>In <a>scripts</a> that have very little variation in source language,
+                    for example audio description scripts,
+                    the <a><code>daptm:langSrc</code></a> attribute can be set on the root element
+                    and omitted from <code>&lt;p&gt;</code> elements except where its value
+                    differs.
```

Comment:
I wonder if audio description scripts are the best example for this use case. In most cases audio description scripts would be descriptions of scenes in the programme audio where the source has no inherent language. Given the current PR text it would not be needed to specify `@daptm:langSrc`. 

(But see also proposal to disallow the attribute on `<tt>`, `<body>` and `<div>` in my comment https://github.com/w3c/dapt/pull/179#issuecomment-1859142948)

Reply:
Right, can just delete the "for example audio description scripts," line.